lib/MusicBrainz/Server/dbmirror.pm

```diff
@@ -3,6 +3,8 @@
 use warnings;
 use strict;
 
+use English;
```

Comment:
This commit looks like a mistake, since `use English;` is already declared below (or it should be merged into the previous commit if it should be moved).

Reply:
Yeah, that's a mistake on rebasing, thanks for catching it!